### PR TITLE
Fix for win10

### PR DESCRIPTION
**Pull request by asika32764**
For issue https://github.com/aik099/PhpStormProtocol/issues/24

I can make sure this change works fine in my Win10 computer, but I can't test it with Win7, 8 and other versions.

### CODE REVIEW

File: PhpStorm Protocol (Win)/run_editor.js [hunk 1]
```diff
@@ -17,9 +17,9 @@ var settings = {
 
 // don't change anything below this line, unless you know what you're doing
 var	url = WScript.Arguments(0),
-	match = /^phpstorm:\/\/open\?(url=file:\/\/|file=)(.+)&line=(\d+)$/.exec(url),
+	match = /^phpstorm:\/\/open\/\?(url=file:\/\/|file=)(.+)&line=(\d+)$/.exec(url),
 	project = '',
-	editor = '"C:\\' + (settings.x64 ? 'Program Files (x86)' : 'Program Files') + '\\JetBrains\\' + settings.folder_name + '\\bin\\PhpStorm.exe"';
+	editor = '"C:\\' + (settings.x64 ? 'Program Files (x86)' : 'Program Files') + '\\JetBrains\\' + settings.folder_name + '\\bin\\phpstorm.exe"';
```
Comment (aik099) [inline PhpStorm Protocol (Win)/run_editor.js]: Are you sure, that binary file is named differently now?

File: PhpStorm Protocol (Win)/run_editor.js [hunk 2]
```diff
@@ -3,10 +3,10 @@ var settings = {
 	x64: true,
 
 	// Set to folder name, where PhpStorm was installed to (e.g. 'PhpStorm')
-	folder_name: 'PhpStorm 2016.1.2',
+	folder_name: 'PhpStorm 2016.3',
```
Comment (aik099) [inline PhpStorm Protocol (Win)/run_editor.js]: Actually it's 2017.something now. Please update.

File: PhpStorm Protocol (Win)/run_editor.js [hunk 3]
```diff
@@ -3,10 +3,10 @@ var settings = {
 	x64: true,
 
 	// Set to folder name, where PhpStorm was installed to (e.g. 'PhpStorm')
-	folder_name: 'PhpStorm 2016.1.2',
+	folder_name: 'PhpStorm 2016.3',
 
 	// Set to window title (only text after dash sign), that you see, when switching to running PhpStorm instance
-	window_title: 'PhpStorm 2016.1.2',
+	window_title: 'PhpStorm 2016.3',
```
Comment (aik099) [inline PhpStorm Protocol (Win)/run_editor.js]: Actually it's 2017.something now. Please update.

File: PhpStorm Protocol (Win)/run_editor.js (hunk 1 above)
Comment (asika32764) [inline PhpStorm Protocol (Win)/run_editor.js]: It's lowercase in my computer, see:

p-2017-06-30-001
p-2017-06-30-002

It seems phpstorm's binary name and folder name are different in everyone's computer, so I think we need more people's test.

Comment (aik099) [inline PhpStorm Protocol (Win)/run_editor.js]: I guess folder name is created based on what version you've installed initially and then stays unchanged after upgrades. We'll keep it as is to be sure.

**Comment by aik099** [comment]
Merging. Thanks, @asika32764 .

**Pull request merged**